src/js/utils/dom.js

```diff
@@ -728,6 +728,52 @@ export function insertContent(el, content) {
 }
 
 /**
+ * Check if event was a single left click
+ *
+ * @param {EventTarget~Event} event
+ *        Event object
+ *
+ * @return {boolean}
+ *         - True if a left click
+ *         - False if not a left click
+ */
+export function isSingleLeftClick(event) {
+  /* Note: if you create something draggable, be sure to
```

Comment:
Minor stylistic concern, but could we stick to using `/* */` comments for JSDoc only and use `//` for all inline comments (even if they span multiple lines)?